cmds/core/shutdown/shutdown_linux.go

```diff
@@ -58,7 +58,7 @@ func shutdown(dryrun bool, args ...string) (uint, error) {
 	}
 	op, ok := opcodes[args[0]]
 	if !ok {
-		return 0, fmt.Errorf(usageMessage)
+		return 0, errors.New(usageMessage)
```

Comment:
nit: instead of creating error here it better to replace `const usageMessage` with `var errUsage = errors.New("shutdown [<-h|-r|-s|halt|reboot|suspend> [time [message...]]]")` it may simplify testing code

Reply:
This sounds very reasonable, I will do it like that.